README.md

```diff
@@ -270,11 +264,18 @@ Zarf is written entirely in [go](https://go.dev/), except for a single 400Kb bin
 - Currently the Registry and Git servers _are not HA_, see [#375](https://github.com/defenseunicorns/zarf/issues/376) and [#376](https://github.com/defenseunicorns/zarf/issues/376) for discussion on this
 - In order to avoid TLS issues, Zarf binds to `127.0.0.1:319999` on each node as a [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport) to allow all nodes to access the pod(s) in the cluster
 - Until [#306](https://github.com/defenseunicorns/zarf/pull/306) is merged, during helm install/upgrade a [Helm PostRender](https://helm.sh/docs/topics/advanced/#post-rendering) function is called to mutate images and [ImagePullSecrets](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod) so the deployed resoures use the NodePort binding
-- Once [#329](https://github.com/defenseunicorns/zarf/pull/329) is merged, Zarf will use a new injector system to bootstrap a new cluster:
+- With [#329](https://github.com/defenseunicorns/zarf/pull/329), Zarf uses a new injector system to bootstrap a new cluster:
```

Comment:
I don't think the main README is the best place to document stuff like this. General 'nerd notes' is great and I can understand having a few 'coming soon' messages saying what interesting upcoming issues/PRs are tackling. But saying 'With this specific PR you can do this now' seems a lot more like something that belongs in tagged release notes than something that belongs in a README.